internal/dbutil/pgutil/db.go

```diff
@@ -98,3 +100,17 @@ func QuerySnapshot(db dbschema.Queryer) (*dbschema.Snapshot, error) {
 		Data:    data,
 	}, err
 }
+
+//CheckApplicationName ensures that the Connection String contains an application name
+func CheckApplicationName(s string) (r string) {
+	if !strings.Contains(s, "application_name") {
+		if !strings.Contains(s, "?") {
+			r = fmt.Sprintf("%s?%s", s, "application_name=Satellite")
+		} else {
+			r = fmt.Sprintf("%s%s%s", s, "&", "application_name=Satellite")
```

Comment:
nit:
you could really just use string concatenation here and above.
you could also directly return here and above.

Reply:
Sure, let me change that real quick!